### PR TITLE
Add OpenShift

### PR DESCRIPTION
**Pull request by ritawatson**
This PR adds OpenShift to the list of supported platforms and adds a tab with steps on how to deploy a virtual cluster in OpenShift.

Jira: https://spectrocloud.atlassian.net/browse/PPD-820

[Preview](https://deploy-preview-1037--docs-spectrocloud.netlify.app/devx/cluster-groups)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1037#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (TylerGillson) [inline content/docs/04.5-devx/03-cluster-groups.md:114]: The cluster group doesn't necessarily have to be new. Maybe this should say, `Create a new Cluster Group or edit an existing Cluster Group and click **Settings**.`

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (TylerGillson) [inline content/docs/04.5-devx/03-cluster-groups.md:120]: It wouldn't hurt to be 100% explicit, e.g.:
`fsGroup: 12345`
`runAsGroup: 12345`
`runAsUser: 12345`

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (TylerGillson) [inline content/docs/04.5-devx/03-cluster-groups.md:124]: This step is a non-step (no action required) so maybe we can leave it out. Or if we include it, we should also mention
```
capabilities:
  drop:
  - all
```

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md:120]: @TylerGillson That's strange, the doc source file doesn't specify 12345 at all. It just lists the parameter names - not sure where it's picking up 12345. I've added the value shown in the image: 1001100000

Comment (TylerGillson) [reply content/docs/04.5-devx/03-cluster-groups.md:120]: The defaults that we set are all `12345`

Comment (TylerGillson) [reply content/docs/04.5-devx/03-cluster-groups.md:120]: `1001100000` came from my screenshot in Jira, which is not reflective of the correct default values.

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md]: Updated.

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md]: @TylerGillson I tried creating a new cluster group, and the default is 12345. But since this is for new and existing cluster groups, I really don't want to include a value. The doc just says to comment out those lines.

**Comment by resmaa** [comment]
@ritawatson, Can we add the attached snapshot as code snippet in yaml format

```yaml
file content
```

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md:123]: Thanks @TylerGillson - I'll make that change.

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md]: Thanks for these tips, @TylerGillson. I've incorporated the changes. Could you please double check and let me know if any more changes are needed?

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md]: @TylerGillson Thanks for the tip! Fixed.

**Comment by ritawatson** [comment]
@TylerGillson I fixed inline code format and include a yaml code snippet to replace image. Good suggestion!

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (TylerGillson) [inline content/docs/04.5-devx/03-cluster-groups.md:131]: @ritawatson it should be `capabilities.drop: [all]`, not `capabilities.drop: [-all]`

Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md:131]: @TylerGillson I think I get it. The yaml shows -all, but this is probably just one item in a list of potential other values. That's why I included the dash.

**Review by TylerGillson — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1037#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1037#pullrequestreview-1218108426" title="Approved">TylerGillson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/03-cluster-groups.md:134]:
```suggestion
The example shows the lines to comment out, the lines to verify, and the line to enable OpenShift support.
```

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/03-cluster-groups.md:118]: We should not use a comment as an anchor point as these can change between releases. We are better off pointing to the actual parameter.

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/03-cluster-groups.md:106]: Is the idea to add other platforms in the future?  I guess the first question is, how much different are the other platforms? It just feels a bit awkward with a single tab.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md:107]: fixed

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md:134]: fixed

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md:106]: @karl-cardenas-coding As we discussed, I removed single tab and made it a subsection.

File: content/docs/04.5-devx/03-cluster-groups.md
Comment (ritawatson) [reply content/docs/04.5-devx/03-cluster-groups.md:118]: @karl-cardenas-coding The doc now refers to the SecurityContext section.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1037#pullrequestreview-1218223878" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1037#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1037#pullrequestreview-1218108426" title="Approved">TylerGillson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
